modules/ossm-creating-certificates-for-multi-cluster-mesh.adoc

```diff
@@ -0,0 +1,197 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-creating-certificates-for-a-multi-cluster-mesh_{context}"]
+= Creating certificates for a multi-cluster mesh 
+
+This procedure explains how to create root and intermediate certificates for two clusters, named East and West.
+
+[NOTE]
+====
+In this procedure, `cluster1` is the East cluster and `cluster2` is the West cluster. 
+====
```

Comment:
This note isn't needed, because the terms "cluster1" or "cluster2" never appear in this document.

Reply:
done